code/modules/mob/living/simple_animal/hostile/syndicate.dm

```diff
@@ -160,4 +161,9 @@
 
 /mob/living/simple_animal/hostile/viscerator/death()
 	..(null,"is smashed into pieces!")
+	var/T = get_turf(src)
+	new /obj/effect/gibspawner/robot(T)

```

Comment:
I approve of this, but it might be good to modify the New of gibspawners with some amount/mass input, or otherwise make it check the mob size of the mob in question, so that these little things don't make as much mess as a 300kg cyborg